### DISCUSSION TITLE
Feature-Sliced Design Nuxt 3

### DISCUSSION THREAD

**Discussion by Firstasianinspace** [nuxt/nuxt#24550] [Questions]
Hello everyone!
I am just curious how Nuxt projects fit into FSD way.
I didn't find any examples including Nuxt - https://feature-sliced.design/examples
Can you share you thoughts about folder structure if i want to try Nuxt in FSD :)
Thank you for visiting this discussion

**Comment by MoNSTRiKcom** [comment]
hi! 
Our solution: [nuxt3-feature-sliced-design](https://github.com/e-zybkin/nuxt3_feature-sliced-design)

**Reply by yunglocokid** [reply]
https://github.com/yunglocokid/FSD-Pure-Nuxt3-Template

**Comment by aabounegm** [comment]
I just released a module (not a template) right now for adding support for FSD in Nuxt 3: https://github.com/aabounegm/nuxt-fsd
It's still experimental (I didn't test it thoroughly), but any feedback is appreciated!
